docs/API.md

```diff
@@ -1193,11 +1193,14 @@ can call:
     redraws (e.g. from a mousemove). You can get to the overlay by
     setting up a drawOverlay hook.
     This function will also trigger the dispatching of the
-    'onDrawingDone' custom event, which can be used to notify listeners
-    (drawOverlay hooks) about the finishing of a drawing operation.
+    'onDrawingDone' custom event on the eventholder, and "drawingdown"
+    event on the placeholder, which can be used to notify listeners
+    about the finishing of a drawing operation (drawOverlay hooks).
     Use addEventListener('onDrawingDone', callback) and
-    removeEventListener('onDrawingDone', callback), to register and unregister
-    handlers for 'onDrawingDone' event.
+    removeEventListener('onDrawingDone', callback), to register and
+    unregister handlers for 'onDrawingDone' event, or use
+    on('drawingdone', callback) and off('drawingdone', callback)
```

Comment:
Where does this recommendation to use 'on' and 'off' come from? The docs suggest patterns like the [following](https://github.com/flot/flot/blame/master/docs/API.md#L1030) for attaching event listeners to the placeholder object.

Reply:
Changed to `bind` and `unbind`.